### PR TITLE
Add niftiheader package

### PR DESCRIPTION
**Pull request by WubiCookie**

### CODE REVIEW

File: packages/n/niftiheader/xmake.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package("niftiheader")
+    set_kind("library", {headeronly = true})
+    add_urls("https://nifti.nimh.nih.gov/pub/dist/src/nifti2/")
+    add_versions("latest", "")
+
+    on_download(function(package, opt)
+        import("net.http")
+        http.download(opt.url .. "nifti1.h", path.join(opt.sourcedir, "nifti1.h"))
+        http.download(opt.url .. "nifti2.h", path.join(opt.sourcedir, "nifti2.h"))
+    end)
```
Comment (waruqi) [inline packages/n/niftiheader/xmake.lua:10]: you can remove on_download and add_urls to. https://github.com/xmake-mirror/nifti2 mirror

Comment (WubiCookie) [reply packages/n/niftiheader/xmake.lua:10]: I don't understand, the link you posted returns 404 Not Found

Comment (waruqi) [reply packages/n/niftiheader/xmake.lua:10]: 404?

image

Comment (waruqi) [reply packages/n/niftiheader/xmake.lua:10]:
```lua
add_urls("https://github.com/xmake-mirror/nifti2.git")
add_versions("0.0.1", "d2461b27888fd5db659edb8b788c7ce6203f703c")
```

then remove on_download

Comment (WubiCookie) [reply packages/n/niftiheader/xmake.lua:10]:
```
$ git clone https://github.com/xmake-mirror/nifti2.git
Cloning into 'nifti2'...
remote: Repository not found.
fatal: repository 'https://github.com/xmake-mirror/nifti2.git/' not found
```
I don't seem to have access to this repo...

Comment (WubiCookie) [reply packages/n/niftiheader/xmake.lua:10]: oh, it is private indeed
image

Comment (waruqi) [reply packages/n/niftiheader/xmake.lua:10]: oh, sorry, I have made it as public.

File: packages/n/niftiheader/xmake.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package("niftiheader")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://nifti.nimh.nih.gov/")
```
Comment (waruqi) [inline packages/n/niftiheader/xmake.lua:3]: please add `set_description` and `set_license`

Comment (waruqi) [reply packages/n/niftiheader/xmake.lua:3]: and add a new line after `set_license`, you can refer other packages in xmake-repo.

**Pull request merged**